app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/ViewAnnotationShowcaseActivity.kt

```diff
@@ -0,0 +1,197 @@
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import android.util.TypedValue
+import android.view.View
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.asynclayoutinflater.view.AsyncLayoutInflater
+import com.mapbox.geojson.Feature
+import com.mapbox.geojson.FeatureCollection
+import com.mapbox.geojson.Point
+import com.mapbox.maps.*
+import com.mapbox.maps.extension.style.image.image
+import com.mapbox.maps.extension.style.layers.generated.symbolLayer
+import com.mapbox.maps.extension.style.layers.properties.generated.IconAnchor
+import com.mapbox.maps.extension.style.sources.generated.GeoJsonSource
+import com.mapbox.maps.extension.style.sources.generated.geoJsonSource
+import com.mapbox.maps.extension.style.sources.generated.rasterDemSource
+import com.mapbox.maps.extension.style.sources.getSourceAs
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.extension.style.terrain.generated.terrain
+import com.mapbox.maps.plugin.gestures.*
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.databinding.ActivityViewAnnotationShowcaseBinding
+import com.mapbox.maps.viewannotation.ViewAnnotationManager
+import com.mapbox.maps.viewannotation.viewAnnotationOptions
+
+/**
+ * Example how to add view annotations to the map.
+ *
+ * Specifically view annotations will be associated with marker icons
+ * showcasing how to implement functionality similar to MarkerView from Maps v9.
+ */
+class ViewAnnotationShowcaseActivity : AppCompatActivity(), OnMapClickListener, OnMapLongClickListener {
+
+  private lateinit var mapboxMap: MapboxMap
+  private lateinit var viewAnnotationManager: ViewAnnotationManager
+  private lateinit var style: Style
+  private val pointList = mutableListOf<Feature>()
+  private var markerId = 0
+
+  private var markerWidth = 0
+  private var markerHeight = 0
+
+  private val asyncInflater by lazy { AsyncLayoutInflater(this) }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    val binding = ActivityViewAnnotationShowcaseBinding.inflate(layoutInflater)
+    setContentView(binding.root)
+
+    viewAnnotationManager = binding.mapView.viewAnnotationManager
+
+    val bitmap = BitmapFactory.decodeResource(resources, R.drawable.blue_marker_view)
+    markerWidth = bitmap.width
+    markerHeight = bitmap.height
+
+    mapboxMap = binding.mapView.getMapboxMap().apply {
+      loadStyle(
+        styleExtension = prepareStyle(Style.MAPBOX_STREETS, bitmap)
+      ) {
+        style = it
+        addOnMapClickListener(this@ViewAnnotationShowcaseActivity)
+        addOnMapLongClickListener(this@ViewAnnotationShowcaseActivity)
+        binding.fabStyleToggle.setOnClickListener {
+          when (getStyle()?.styleURI) {
+            Style.MAPBOX_STREETS -> loadStyle(prepareStyle(Style.SATELLITE_STREETS, bitmap))
```

Comment:
```suggestion
            MAPBOX_STREETS -> loadStyle(prepareStyle(SATELLITE_STREETS, bitmap))
```
use import for this styles

Reply:
across the project `Style.` is used, let's be consistent